src/components/ExternalLinkIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function ExternalLinkIcon() {
+  return (
+    <svg width="13.5" height="13.5" aria-hidden="true" style={{ marginLeft: "0.3rem" }}><use href="#theme-svg-external-link"></use></svg>
```

Comment:
Re-use an existing icon component or make this one standalone. Avoid cross-referencing as there is no guarantee the reference will keep working across Docusaurus updates.

Additionally make sure to properly format the file.

Reply:
Done. What to use for formatting? There doesn't seem any check in place to run locally.